clients/client-python/build.gradle.kts

```diff
@@ -122,7 +124,7 @@ fun generatePypiProjectHomePage() {
     // relative path of the images in the how-to-use-python-client.md file is incorrect. We need
     // to fix the relative path of the images/markdown to the absolute path.
     val content = outputFile.readText()
-    val docsUrl = "https://datastrato.ai/docs/latest"
+    val docsUrl = "https://gravitino.apache.org/docs/$gravitinoVersion"
```

Comment:
Please change to
```
val docsUrl = "https://gravitino.apache.org/docs/latest"
```

Reply:
If using latest, all release pypi document will link to the latest Gravitino document page, is it expected?